.github/workflows/ci-postgres-mysql.yml

```diff
@@ -92,7 +90,7 @@ jobs:
 
       - name: Test MySQL
         working-directory: packages/cli
-        run: pnpm test:mysql
+        run: pnpm test:mysql --testTimeout 20000
```

Comment:
I've seen this timeout on `master` at least twice last week. Can you make this PR against `master` instead please 🙏🏽

Reply:
Here you go: https://github.com/n8n-io/n8n/pull/9332